ceno_zkvm/src/instructions/riscv/div.rs

```diff
@@ -179,7 +483,7 @@ mod test {
             circuit_builder::{CircuitBuilder, ConstraintSystem},
             instructions::{
                 Instruction,
-                riscv::{constants::UInt, div::DivUInstruction},
```

Comment:
We can do that rename in its own PR, to make this one easier to review:

https://github.com/scroll-tech/ceno/pull/766